src/game/Com/Com.cpp

```diff
@@ -164,13 +164,13 @@ void Com_AddStartupCommands()
 //THUNK : 0x402500
 void Com_Printf(std::uint32_t channel, char* Format, ...)
 {
-    char Buffer[4096];
+    char Buffer[0x1000];
     va_list ArgList;
```

Comment:
buffer -> msg



> va_list variable name is defined as `va` in IW4 Xenon dedi, `ap` in T6 dedi. I think one of these names are more fitting

It is named argptr and the buffer is named msg

```
Data : cl Relative, [00000050], Local, Type: char *, argptr
Data : cl Relative, [00000060], Local, Type: char[0x1000], msg
```

Reply:
It is not very wise to display so confidently information from pdbs. Other reverse engineering projects would not ever do this.

Reply:
> It is not very wise to display so confidently information from pdbs. Other reverse engineering projects would not ever do this.

https://github.com/diasurgical/devilution

Reply:
Erm what's this supposed to prove? You showed me a repository of a game that was reversed engineered without any external help. It does not also prove that because one company did not care about this project it does not mean that another company let's say acti won't care as well.

Reply:
> Erm what's this supposed to prove? You showed me a repository of a game that was reversed engineered without any external help. It does not also prove that because one company did not care about this project it does not mean that another company let's say acti won't care as well.

That's exactly what this is, another reverse engineering project that, surprise, uses symbolic information. The very thing you said and I quote, "Other reverse engineering projects **would never do this**."

However, if you want to dwell on whether or not a company will pay attention to it, that's a whole other topic to discuss and it comes back to the same gray area of legality of this repository with or without symbolic information. OpenIW4 is not, after all, a clean room design (even that would be legally grey.)